tests/ert_tests/gui/ertwidgets/test_ertsummary.py

```diff
@@ -0,0 +1,40 @@
+import inspect
+from unittest.mock import MagicMock
+
+import pytest
+
+from ecl.util.util import StringList
+from ert_gui.ertwidgets.models.ertsummary import ErtSummary
+
+
+@pytest.fixture
+def mock_ert_summary(monkeypatch, autouse=True):
+    ERT_mock = MagicMock()
+    test_module = inspect.getmodule(ErtSummary)
+    monkeypatch.setattr(test_module, "ERT", ERT_mock)
+
+    string_list = StringList(["forward_model_1", "forward_model_2"])
+    ERT_mock.ert.getModelConfig.return_value.getForwardModel.return_value.joblist.return_value = (
+        string_list
+    )
+
+    string_list = StringList(["param_1", "param_2"])
+    ERT_mock.ert.ensembleConfig.return_value.getKeylistFromVarType.return_value = (
+        string_list
+    )
+
+
+def test_getForwardModels(mock_ert_summary):
```

Comment:
As the fixture is a pure set up fixture where you dont use a return value you _could_ do: 
```
@pytest.mark.usefixtures("mock_ert_summary")
```
and even that is not needed as you are using `autouse`, though personally I tend to stay away from that as it is a bit too magic for my taste.

Reply:
Ah, that looks much better, thanks.